### ISSUE TITLE
Add images to wiki

### ISSUE THREAD

**Issue by dougburks** [Security-Onion-Solutions/security-onion#873]
![sodia-bestpractices](https://cloud.githubusercontent.com/assets/1659467/13701945/2f04525c-e759-11e5-8d3d-7973a3f723b6.png)

**Comment by karolisc** [comment]
Hi Doug, Wes,


Put name or url above/below squert and elsa pictures. Just idea maybe
frontend pictures could be aligned?

Move word "sensor " to the bottom right corner of the box like the "Master
server", although i think they both belong to the right top corner of the
box.

Elsa could show index (sphinx) DB's as well.

p.s.
where is the source of this diagram?

Karolis

**Comment by dougburks** [comment]
Tagging @weslambert

**Comment by weslambert** [comment]
Karolis,

Thanks for the suggestions!

I can certainly look into adding the names and URLs for each of the
front-ends.  As far as aligning them, one of the reasons I did not have
them aligned is to prevent the diagram from expanding so far outward.
However, I could try some restructuring and see how it looks.

I could also try playing with each of the titles for the master/sensor to
see if I can make them a little more congruous in terms of
styling/placement.

In regard to the ELSA/Sphinx DB (s), I had to restrict it to a certain
degree in order to prevent crowding.  This is the same case for a few other
components as well.  We could always build out a separate diagram to
illustrate applications/processes in a more in - depth sense.

In all, I wanted to give analysts a basic tool to help them understand the
layout and the flow of the various Security Onion components in regard to
best practices. As, I mentioned we could always elaborate with other, more
specialized diagrams/illustrations in the future.

Again, thank you for your comments!  They are definitely  appreciated!

Thanks,
Wes
On Mar 11, 2016 9:53 AM, "Doug Burks" <notifications@github.com> wrote:

Tagging @weslambert <https://github.com/weslambert>

—
Reply to this email directly or view it on GitHub
<https://github.com/Security-Onion-Solutions/security-onion/issues/873#issuecomment-195399095>
.
